nucypher/blockchain/eth/sol/source/contracts/MinersEscrow.sol

```diff
@@ -1165,15 +1163,12 @@ contract MinersEscrow is Issuer {
         }
     }
 
-    function verifyState(address _testTarget) public onlyOwner {
+    function verifyState(address _testTarget) public {
```

Comment:
Curious - What motivated you to make this method public?

Reply:
Might be because `super.verifyState(_testTarget);` has this modifier, and setting the same modifier several times eats gas (solidity wouldn't optimize this). Similar to the same question in Issuer.sol.

Reply:
Actually, I think, this should be emphasized in a comment: a random reader or auditor would indeed have this question

Reply:
I added comments about `onlyWhileUpgrading` modifier